cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -272,6 +280,10 @@ objects:
             objectName: ${FPA_CERT_NAME}
             objectType: secret
             objectAlias: firstPartyApplicationCertificateBundle
+          - |
```

Comment:
Do we want to approach it having a single SecretProviderClass with N secrets or N SecretProviderClass ? I see different approaches depending on the MR.

Reply:
Both approaches are valid but I opted for this because I see no difference in terms of configuration between the `SecretProviderClass` I needed for this purpose and the firstPartyApp one.  I'll defer this question to @janboll @geoberle on how they see it.